integration_tests/CMakeLists.txt

```diff
@@ -356,14 +356,14 @@ RUN(NAME test_tuple_02       LABELS cpython llvm c)
 RUN(NAME test_tuple_03       LABELS cpython llvm c)
 RUN(NAME test_dict_01        LABELS cpython llvm c)
 RUN(NAME test_dict_02        LABELS cpython llvm c)
-RUN(NAME test_dict_03        LABELS cpython llvm)
+RUN(NAME test_dict_03        LABELS cpython llvm c)
```

Comment:
Let's comment this out, get the CI passing, merge. Open up an issue to investigate what is going on.